### ISSUE TITLE
PocketBase became unresponsive

### ISSUE THREAD

**Issue by FlixMa** [pocketbase/pocketbase#935]
I have my pocketbase instance running perfectly since a month or so (last restart was 2022-10-25).
Unfortunately the process became unresponsive today.

It won't serve any http requests ...
```curl
curl -vv 127.0.0.1:8090/_
*   Trying 127.0.0.1:8090...
* TCP_NODELAY set
* Connected to 127.0.0.1 (127.0.0.1) port 8090 (#0)
> GET /_ HTTP/1.1
> Host: 127.0.0.1:8090
> User-Agent: curl/7.68.0
> Accept: */*
> 
^C
```

... it doesn't react to SIGINTs (`kill <pid>`) and it neither writes to stdout nor stderr. The last entries in error log are long ago:

```
2022/10/02 21:56:47 database is locked (5) (SQLITE_BUSY)
2022/10/02 22:06:48 database is locked (5) (SQLITE_BUSY)
2022/10/02 22:48:43 database is locked (5) (SQLITE_BUSY)
2022/10/09 10:28:54 database is locked (5) (SQLITE_BUSY)
2022/10/09 22:04:54 database is locked (5) (SQLITE_BUSY)
2022/10/22 17:45:21 database is locked (5) (SQLITE_BUSY)
```

The space left on device is fine:

```sh
df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/sda2      1008G  111G  846G  12% /
```

CPU is almost idling and RAM usage is about 2/3.

Any ideas what I can check?
Is my only option killing it and restarting using debug flag hoping this yields more information next time?

**Comment by FlixMa** [comment]
While trying to find out the version I am using, other processes were starting to  misbehave as well. Now my I am getting the following error when trying to write data: `mkdir: cannot create directory ‘pb_bak’: Read-only file system`
Sorry for bothering

**Comment by ganigeorgiev** [comment]
@FlixMa There were several improvements in the latest releases (including a fix for the `SQLITE_BUSY` error in https://github.com/pocketbase/pocketbase/issues/875).

If you are using PocketBase v0.7.x, you could try upgrading to the latest [v0.7.10](https://github.com/pocketbase/pocketbase/releases/tag/v0.7.10).

A lot of other improvements were also done but they are available only in the pre-release [v0.8.0-rc](https://github.com/pocketbase/pocketbase/releases/tag/v0.8.0-rc2).
